files/build_templates/buffers_config.j2

```diff
@@ -133,6 +134,9 @@ def
         {%- if not port.startswith('Ethernet-Rec') and not port.startswith('Ethernet-IB') %}
             {%- if PORT_ALL.append(port) %}{%- endif %}
         {%- endif %}
+        {%- if port.startswith('Ethernet-BP') %}
```

Comment:
this might effect other multi-asic platforms that also have port names starting with Ethernet-BP (refer to L100)

Reply:
@neethajohn I have made an update, please review